lib/core/test/unit/Cardano/Pool/DB/Properties.hs

```diff
@@ -188,6 +193,8 @@ properties = do
             (property . prop_rollbackRegistration)
         it "rollback of PoolRetirement"
             (property . prop_rollbackRetirement)
+        it "removePools"
+            (property . prop_removePools)
```

Comment:
Still no state machine tests for stake pools?

Reply:
> Still no state machine tests for stake pools?

I agree this is something we should definitely add.

I've added an item of technical debt here: https://jira.iohk.io/browse/ADP-406